converter/internal/staticconvert/internal/build/builder.go

```diff
@@ -98,3 +155,11 @@ func (b *IntegrationsV1ConfigBuilder) appendExporter(commonConfig *int_config.Co
 	b.diags.AddAll(prometheusconvert.AppendAllNested(b.f, promConfig, jobNameToCompLabelsFunc, extraTargets, b.globalCtx.RemoteWriteExports))
 	b.globalCtx.InitializeRemoteWriteExports()
 }
+
+func splitByCommaNullOnEmpty(s string) []string {
```

Comment:
Sometimes static mode is using comma separated lists such as windows exporter enabled collectors. This helper func will save us from putting empty arrays on the river output by returning nil for an empty string.